### PR TITLE
[Build apps for Teams meetings][Doc revamp][1700925]

### PR DESCRIPTION
**Pull request by v-pmuthyala**
Added note to side panel

### CODE REVIEW

**Review by SubbaReddi — APPROVED** [review]
Looks fine to me. Thanks

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -316,6 +316,9 @@ To use the `userContext` API to route requests accordingly, see [Teams SDK](../t
 
 Messaging extension works as expected when a user is in an in-meeting view and the user can post compose message extension cards. AppName in-meeting is a tooltip that states the app name in-meeting U-bar.
 
+> [!NOTE]
+> * The side panel is not supported in the older version of SDK. Make sure you use the latest version 1.9.0 of [Teams SDK](https://docs.microsoft.com/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true). The minimum required version is 1.9.0.
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:320]: Please rephrase the note to remove repetitive information and remove the *.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**